pylint/checkers/method_args.py

```diff
@@ -71,13 +71,11 @@ class MethodArgsChecker(BaseChecker):
     )
     def visit_call(self, node: nodes.Call) -> None:
         self._check_missing_timeout(node)
-        self._check_positional_only_arguments_expected(node)
+        # Other checks...
```

Comment:
done

Reply:
I think you added a python file in the documentation, but you need to modify the code / add your logic here:

https://github.com/pylint-dev/pylint/blob/27b1ae7f8dfeedcc817ca8506b5835c468ebc658/pylint/checkers/method_args.py#L76-L100

Please note that we're using inference which can handle case where a variable is used and not ``request.Session`` directly.

Reply:
but in the main branch is it showing inference which can handle case where a variable is used and not request.Session directly. or is it my mistake

Reply:
Hmm, not sure. I think we handle a call to request.get (and other see ``self.linter.config.timeout_methods``)  (directly or inferred) but not Session. It should be able to do both (?). Maybe the solution is as simple as changing the default for ``timeout_methods``

Reply:
i did the change a bit can you is it working

Reply:
One way to test this would be to create a functional tests, see here: https://pylint.readthedocs.io/en/stable/development_guide/contributor_guide/tests/writing_test.html#functional-tests and here https://github.com/pylint-dev/pylint/blob/main/tests/functional/m/missing/missing_timeout.py